app/styles/components/_offering-editor-learnergroups.scss

```diff
@@ -0,0 +1,40 @@
+@mixin subgroup-style {
```

Comment:
I made modifications to `_tree-list.scss` so the style looks consistent, but `subgroup-style` needs to remain in here because the structure for `offering-editor-learnergroups` is not consistent with others. It uses one layer of `li` tags , instead of two layers.